fdbserver/LogSystemConfig.h

```diff
@@ -292,6 +292,14 @@ struct LogSystemConfig {
 		return results;
 	}
 
+	int numLogs() const {
```

Comment:
This will include remote tlogs. For the purpose of TPCV, we only need to know **local** tlogs, i.e., `tLogs[i].isLocal == true`, because proxy won't write to remote tlogs.

Reply:
Fixed